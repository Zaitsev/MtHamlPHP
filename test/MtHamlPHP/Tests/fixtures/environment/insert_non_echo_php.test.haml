%x{$foo => "a #{$b} #{$c + $d}", $bar => 1}
