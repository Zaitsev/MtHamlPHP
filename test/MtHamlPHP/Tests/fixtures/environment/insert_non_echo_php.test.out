<x $foo="a <?php echo ( htmlspecialchars($b,ENT_QUOTES,UTF-8)) ;?><?php echo ( htmlspecialchars($c + $d,ENT_QUOTES,UTF-8)) ;?>" $bar="<?php echo ( htmlspecialchars(1,ENT_QUOTES,UTF-8)) ;?>"></x>
